meinberlin/apps/kiezradar/templates/meinberlin_kiezradar/search_profile_list.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load static i18n react_kiezradar_search_profiles %}
+
+{% block title %}{% translate 'Search profiles' %} - {{ block.super }}{% endblock title %}
+
+{% block extra_js %}
+    <script type="text/javascript" src="{% static 'react_kiezradar_search_profiles.js' %}"></script>
+    {{ block.super }}
+{% endblock extra_js %}
+
+{% block breadcrumbs %}
+{#    <div id="content-header">#}
```

Comment:
do we have a reminder / task somewhere to add proper breadcrumbs so we don't forget?